src/diffpy/labpdfproc/tools.py

```diff
@@ -0,0 +1,7 @@
+from pathlib import Path
+
+
+def set_output_directory(args):
```

Comment:
If this is a public function it needs a docstring.  If a private function it needs an underscore in front. I am not sure which, but am leaning towards public as we import it from tools.